Light.json

```diff
@@ -14568,7 +14572,7 @@
           "Defensive Shield",
           "Episode I"
         ],
-        "gametext": "Plays on table. Once per game, may take an Immediate Effect into hand from Reserve Deck; reshuffle. While opponent occupies no battleground systems, Mobilization Points is suspended. 'Missing' on Always Thinking With Your Stomach is treated as 'landspeed = 0 for remainder of turn.'",
+        "gametext": "Plays on table. Always Thinking With Your Stomach, Ice Storm, and Sandwhirl are canceled. Once per game, may take an Immediate Effect into hand from Reserve Deck; reshuffle. Unless opponent occupies a battleground system, Mobilization Points is suspended.",
```

Comment:
Did the Legacy version really have that new text too?  I know the non-legacy one got updated to make those cards canceled, but did the legacy one already have that?